pyemma/coordinates/data/writer.py

```diff
@@ -71,7 +73,7 @@ def _reset(self, stride=1):
             pass
 
         try:
-            self._fh = open(self.filename, 'w')
+            self._fh = open(self.filename, 'wb')
```

Comment:
im not quite sure about this but isn't 'wb' binary mode and CSV is not binary..? I have found [here](http://stackoverflow.com/a/1020077/2871028) that "wb" is correct for python 2.x but "w" should be used for py3k when dealing with csv.